File: content/atlas/source/includes/avs/rag/retrieve/retrieve-documents-test.go
package main

import (
	"fmt"
	"rag-mongodb/common" // Module that contains the GetQueryResults function
)

func main() {
	query := "AI Technology"
	documents := common.GetQueryResults(query)
	for _, doc := range documents {
		fmt.Printf("Text: %s \nScore: %v \n\n", doc.PageContent, doc.Score)
	}
}

File: content/vector-search/source/includes/rag/code-snippets/retrieve/retrieve-documents-test.js
import { getQueryResults } from './retrieve-documents.js';

async function run() {
    try {
        const query = "AI Technology";
        const documents = await getQueryResults(query);

        documents.forEach( doc => {
            console.log(doc);
        }); 
    } catch (err) {
        console.log(err.stack);
    }
}
run().catch(console.dir);
